internal/docgen/parsing.go

```diff
@@ -92,13 +92,15 @@ func semgrepRules(destinationDir string) ([]PatternWithExplanation, *ParsedSemgr
 func getSemgrepRegistryRules() (*ParsedSemgrepRules, error) {
 	return getRules(
 		"https://github.com/semgrep/semgrep-rules",
+		"express-fp",
```

Comment:
we should also be able to checkout from a commit uuid if it's easier

Reply:
I agree. At the moment, the git library we use doesn't provide this option (or I couldn't find it). This is why I decided to go with this version